src/sass/components/_section--features.scss

```diff
@@ -1,3 +1,25 @@
 .section--features {
   padding: 5rem 0;
 }
+
+@media (max-width: 576px) {
+  // For cell phones:
+  //grid for media query
+  @for $i from 1 through 12 {
+    .col-sm-#{$i} {
```

Comment:
mamy bootstrapa w projekcie - ten klasy są zbędne

Reply:
wiem, tylko ze  ustawienie tej klasy tylko jako "sm-(*nr)" w elemencie html'a powoduje ze przypisana jest szerokosc 100% bez wzgledu czy ustawie sm-5 czy sm-6 itd. Nie chcialam manipulowac w pliku bootstrapa, dlatego zrobilam tego grida w taki sposob. Moge ewentualnie dodac inna klase na elemencie html, i przyspisac jej  szerokosc w mediaquery. Czy to bedzie lepsze rozwiazanie?

Reply:
Moim zdaniem lepiej osobno zrobić klasę, stworzyć mediaquery i tam wpisać pożądane wartości.

Reply:
Według mnie nie potrzeba robić żadnych klas osobno ani tworzyć dodatkowego media queries. Miałem podobne zadanie WDP190504-15. Ustawienie tej klasy jako <div class="col-sm-6 col-xl-3"> (wyciągnięte z bootrapa) działa i nie powoduje żadnych błędów.